docs/guide/what-are-signals.md

```diff
@@ -8,35 +8,23 @@ description: An introduction to Signals and why they are used for Form Signals.
 In short, Signals are reactive values that can be used to trigger UI updates or calculate computed data based on
 changes.
 
-There are a few frontend libraries, that use Signals as their state management solution, such
-as [Svelte V5](https://svelte.dev/blog/runes), [SolidJS](https://www.solidjs.com/tutorial/introduction_signals), [Angular](https://angular.io/guide/signals), [Qwik](https://qwik.dev/docs/components/state/), [Preact](https://preactjs.com/guide/v10/signals/)
-and possibly many more.
-Signals are not only a concept used for UI libraries, but can also be used in default javascript code to create reactive
-values.
-The Preact team has built a great solution and exposing their Signals API to the public and even created bindings for
+Several frontend libraries use Signals as their state management solution, including [Svelte V5](https://svelte.dev/blog/runes), [SolidJS](https://www.solidjs.com/tutorial/introduction_signals), [Angular](https://angular.io/guide/signals), [Qwik](https://qwik.dev/docs/components/state/), [Preact](https://preactjs.com/guide/v10/signals/)
+(and possibly many more).
+Signals are not just a concept for; they can also be used in vanilla JavaScript code to create reactive values.
```

Comment:
It feels like there is something missing in this scentence, probably you wanted to write something like this
```suggestion
Signals are not just a concept of UI libraries; they can also be used in vanilla JavaScript code to create reactive values.
```

Reply:
You're right. The sentence should have been:

"Signals are not just a concept used for UI libraries; they can also be used in vanilla JavaScript code to create reactive values."

I'll fix that